sed/core/processor.py

```diff
@@ -92,29 +141,598 @@ def coordinates(self, coords):
         for k, v in coords.items():
             self._coordinates[k] = xr.DataArray(v)
 
-    @config.setter
-    def config(self, config: Union[dict, str]):
-        self._config = parse_config(config)
-        if "num_cores" in self._config["binning"].keys():
-            if self._config["binning"]["num_cores"] >= N_CPU:
-                self._config["binning"]["num_cores"] = N_CPU - 1
-        else:
-            self._config["binning"]["num_cores"] = N_CPU - 1
+    def cpy(self, path: Union[str, List[str]]) -> Union[str, List[str]]:
```

Comment:
Not clear what this cpy method does. I see that it is using the copy tool which I commented on as well

Reply:
The purpose of the copy tool is to transparently mirror data from the source path (typically a network drive) to a local data storage. At least for our data, binning from a local storage is ~10x faster compared to binning across the network. This function here is just a convenient wrapper that uses config entries for instantiation.